app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+list_of_final_object_mut = []
+list_of_final_object_immut = []
+
+sorted_variables = {
+    "mutable": list_of_final_object_mut,
+    "immutable": list_of_final_object_immut
+}
+
+all_objects = [
+    lucky_number, pi, one_is_a_prime_number,
+    name, my_favourite_films, profile_info,
+    marks, collection_of_coins
+]
+
+for obj in all_objects:
+    if isinstance(obj, dict) or isinstance(obj, set) or isinstance(obj, list):
+        list_of_final_object_mut.append(obj)
+    else:
+        list_of_final_object_immut.append(obj)
```

Comment:
Your solution should consist of only the `sorted_variables` dict with two keys: `"mutable"` and `"immutable"`. As values, you should have lists that are hardcoded with variables that refer to either `mutable` or `immutable` data types. See the example in the task description and remove the additional logic that is present here.